spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/metrics/web/reactive/server/MetricsWebFilter.java

```diff
@@ -48,12 +49,46 @@
 
 	private final boolean autoTimeRequests;
 
+	private final double[] autoTimeRequestsDefaultPercentiles;
+
+	private final boolean autoTimeRequestsDefaultHistogram;
+
+	/**
+	 * Create a new {@code MetricsWebFilter}.
+	 * @param registry the registry to which metrics are recorded
+	 * @param tagsProvider provider for metrics tags
+	 * @param metricName name of the metric to record
+	 * @param autoTimeRequests if requests should be automatically timed
+	 * @deprecated since 2.1.4 in favor of
```

Comment:
This PR is intended to be merged on `master` only so that should be deprecated as of `2.2.0`